packages/editor/src/components/use-styles/index.js

```diff
@@ -34,10 +39,10 @@ export function useStyles() {
 		};
 	}, [] );
 
-	// Make sure to recompute the styles when hasResolved changes.
+	// Make sure to update styles when isReady changes.
 	const config = useMemo( () => {
```

Comment:
Asked about use of `useMemo` over in https://github.com/WordPress/gutenberg/pull/59929/files#r1577277375

Reply:
Thanks, I think the need for it was probably reduced further with the introduction of the `DEFAULT_STYLES` constant.

I removed it in https://github.com/WordPress/gutenberg/pull/61022/commits/cc4ac26aa295f95f5a33aee16d20812e328ee85d on this branch.